### PR TITLE
feat: allow shortlink name to open new tab onclick.

### PR DESCRIPTION
**Pull request by JoHeyJo**
- clicking on a link will open the site corresponding to the link in a new tab.
- Remove the default styling of the link to maintain the original style. 
- Add hover effect to indicate the link.

### CODE REVIEW

File: src/popup.tsx [hunk 1]
```diff
@@ -87,7 +87,9 @@ function Popup() {
             <div className="shortlink-container">
               {allShortlinks.map((shortlink) => (
                 <code key={shortlink.name} className="shortlink">
-                  {shortlink.name}
+                  <a href={`https://www.${shortlink.name}`} target="_blank" className="shortlink-link">
```
Comment (nazmulidris) [inline src/popup.tsx:90]: @JoHeyJo This actually does not work. Each shortlink name is a key. The value is a list of URLs which have to be opened when this is clicked.

File: src/style.css [hunk 2]
```diff
@@ -86,3 +86,8 @@ h1 {
   border: var(--default-border-style);
   padding: var(--default-padding);
 }
+
+.shortlink-link{
+  all: unset;
+  cursor: pointer;
+}
```
Comment (nazmulidris) [inline src/style.css:93]: Can you also add an underline effect when the user hovers over a link?

**Review by nazmulidris — CHANGES_REQUESTED** [review]
The code does not actually work. Clicking on a shortlink results in an error. It should resolve the shortlink name into a list of URLs.

File: src/popup.tsx (hunk 1 above)
Comment (JoHeyJo) [reply src/popup.tsx:90]: so every URL in `shorlink.urls` should be opened as a new tab? I'm also noticing the URLs stored in `urls` don't match `shortlink.name`

**Comment by nazmulidris** [comment]
@JoHeyJo  Thank you for spending time on this. Even though we didn't accept your PR, we added `hacktoberfest-accepted` label to it, so you should get credit for it.

File: src/popup.tsx (hunk 1 above)
Comment (nazmulidris) [reply src/popup.tsx:90]: This commit shows what has to happen:  https://github.com/r3bl-org/shortlink/commit/947345e0f9106989303265bc8d71b9e1f16c9046 👍🏽 

The shortlink name is a key. The hashtable that stores the value is chorme.storage.sync. The value for the key is of type URLs.

**Comment by JoHeyJo** [comment]
Got it. Thanks!

**Pull request closed without merging**